tests/ci/base-spec.js

```diff
@@ -101,6 +101,8 @@ describe('CiBase', function () {
   describe('.push()', function () {
     let result
     beforeEach(function () {
+      base.vcs = {addRemoteForPush () {}}
+      sandbox.stub(base.vcs, 'addRemoteForPush').returns(Promise.resolve('origin'))
```

Comment:
A few problems here. First, you don't need to define it as an empty function and then create a stub, you can just define it as a stub. Also, if you want to return a promise that resolves to something, there are [new `sinon` methods](http://sinonjs.org/releases/v2.0.0/stubs/) that make that even easier:
```js
base.vcs = {
  addRemoteForPush: sandbox.stub().resolves('origin')
}
```

Unfortunately, this doesn't actually test what you want it to. With this change to the test, I could revert your change to `lib/ci/base.js` and the test would still pass. 
I'm guessing the workflow you followed was 
 1. making your change
 1. modify the test so that it passes again. 

I suggest you try the following workflow in future: 
 1. Write a failing test
 1. Modify code so the test passes

Or, at the very least
 1. Make your change
 1. Add additional test
 1. Try reverting your change to ensure test fails: